### PR TITLE
Add missing .float() for bf16 inference for FSQ models

### PR DESCRIPTION
**Pull request by NilanEkanayake**
Simple change that allows BF 16 inference like so:
```
model.to(torch.bfloat16)
with torch.no_grad(), torch.autocast(device_type='cuda:0', dtype=torch.bfloat16):
    _, xrec, _ = model(input)
```

### CODE REVIEW

**Comment by deeptimhe** [comment]
Hi, thanks for the contribution! We are working on testing the loss for bfloat16 inference.
